NodeOpera/generate_config

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+OPERA_VERSION=60.0.3255.170
```

Comment:
Why do we need to hardcode the Opera version?

Reply:
This hardcoded version is supported by used opera chromium driver. Should I use different approach?

Reply:
Please check if the same approach used for Chrome works here, which is installing the latest available package from the distribution. 
The idea is that a browser upgrade should be just running a new build.

Reply:
Ok, I removed all hardcoded versions and I made it dynamic based on approach used for Chrome docker images.